### PR TITLE
Feat(migration): Netlify to Amplify migration

### PR DESCRIPTION
**Pull request by kishore03109**
# Problem 
Currently, we do have a script to change permalinks. But, the rest of the steps are done manually. To aid this process, this script is run to migrate the existing websites from netlify to amplify. Websites in netlify is not deleted intentionally in case something goes wrong, we can fall back to netlify. Inclined to keep the sites live (which are redundant) until migration is over and we are more confident that there arent breaking changes. To be tackled when we think about decommissioning netlify as a whole. 

Known (intentional) limitations: 
- This script not directly talking to sql database. As a result, user id is manually hardcoded, and there a need to copy paste sql commands into prod db. Since this migration is only run once per site, I think we shouldnt over-engineer an API endpoint for something that we are not maintaining in the long run. 

## Testing
Since this is infrastructure changes, it is more challenging to test this and capture all edge cases (esp considering the existence of v1 sites). I think decent effort is to manually test if this works. Currently, we have around 50 sites that has been migrated from Netlify to Amplify using this script. In addition to this, changing the permalinks still works on Netlify, ie, the changes are backward compatible, which suggests that we can migrate this silently in the background without affecting user flow. To further improve our confidence in this migration, currently testing this for Customs, Go business, mse.gov.sg (3 big users with quite number of breaking changes to see if there)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +17,14 @@ Ideally, this should be done by the designated team account, not with your perso
 node migrationScript.js <repo name>
 ```
 
+### To migrate from Netlify to Amplify
+
+Prerequisites:
+
+- AWS CLI installed
```
Comment (seaerchin) [inline README.md:24]: think we need the appropriate configuration files too or link to the setup instructions on aws itself for clarity

File: README.md [hunk 2]
```diff
@@ -15,3 +17,14 @@ Ideally, this should be done by the designated team account, not with your perso
...
+- Git
+- Hub CLI
```
Comment (seaerchin) [inline README.md:26]: just double checking - is this intended?

File: bash_scripts/netlify_to_amplify_migration/netlify_to_amplify_migration.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+# Write the modified yaml file back to disk
+echo "$yaml_file" > _config.yml
+
+
+git add _config.yml
+git commit -m "chore: update config.yml file"
+echo "Pushing to remote"
+git push -u origin add-trailing-slash
+#Create pull request
```
Comment (seaerchin) [inline bash_scripts/netlify_to_amplify_migration/netlify_to_amplify_migration.sh:43]: super ultra omega nit: add space

File: bash_scripts/netlify_to_amplify_migration/netlify_to_amplify_migration.sh [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+hub pull-request -b staging -h add-trailing-slash -m "migrate: adding trailing slash to permalinks" -f --no-edit --squash
+
+
+git push origin --delete add-trailing-slash
+
+echo "Merge and delete of add-trailing-slash branch successful"
+
+cd "$current_pwd"
+# generate scipt to update prod db with new values
+user_id=1 #todo figure out how to change this hardcoded value to the actual user id
```
Comment (seaerchin) [inline bash_scripts/netlify_to_amplify_migration/netlify_to_amplify_migration.sh:53]: super ultra omega nit: `# TODO: ` cos some extensions in vscode requiring uppercsae to show the color for keywords

File: bash_scripts/netlify_to_amplify_migration/netlify_to_amplify_migration.sh [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+echo >> sqlcommands.txt
```
Comment (seaerchin) [inline bash_scripts/netlify_to_amplify_migration/netlify_to_amplify_migration.sh:55]: can i check why this file is never used?

**Review by seaerchin — APPROVED** [review]
lgtm, just minor style stuff

**Comment by kishore03109** [comment]
Sorry @seaerchin, didnt realise this PR was still up. Developing in bash was pain, have replaced it with[ ts instead](https://github.com/isomerpages/isomer-conversion-scripts/pull/55). Closing this for now.
